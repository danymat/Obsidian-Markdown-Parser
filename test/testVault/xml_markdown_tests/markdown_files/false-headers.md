Root level text.

# A good header

This header has good text.

#bad header

asdf # another bad header

    ## This is okay

asdfasdf
