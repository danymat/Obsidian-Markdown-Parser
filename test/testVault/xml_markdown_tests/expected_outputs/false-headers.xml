<root level="0">Root level text.

<section level="1" title="A good header"># A good header

This header has good text.

#bad header

asdf # another bad header

<section level="2" title="This is okay">    ## This is okay

asdfasdf
</section></section></root>
